cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -193,6 +196,26 @@ public void execute() {
             return;
         }
 
+        String[] args = new String[0];
+        if (!argList.isEmpty()) {
+            if (!argList.get(0).equals("--")) {
+                this.projectPath = Paths.get(argList.get(0));
+                if (argList.size() > 1 && !argList.get(1).equals("--")) {
+                    CommandUtil.printError(this.errStream,
+                            "unmatched command argument found: " + argList.get(1), testCmd, false);
+                    CommandUtil.exitError(this.exitWhenFinish);
+                    return;
+                }
+                if (argList.size() > 2 && argList.get(1).equals("--")) {
+                    args = argList.subList(2, argList.size()).toArray(new String[0]);
```

Comment:
Shall we rename `args` to something that reflects the purpose of args like `configVars`?